src/World.cpp

```diff
@@ -1059,12 +1060,23 @@ void cWorld::Tick(std::chrono::milliseconds a_Dt, std::chrono::milliseconds a_La
 
 	if (m_WorldAge - m_LastSave > std::chrono::minutes(5))  // Save each 5 minutes
 	{
+		LOG("######## PERIODIC SAVE");
 		SaveAllChunks();
 	}
 
+
 	if (m_WorldAge - m_LastUnload > std::chrono::seconds(10))  // Unload every 10 seconds
 	{
+		LOG("######## %d / %d", GetNumUnusedDiryChunks(), m_UnusedDirtyChunksCap);
 		UnloadUnusedChunks();
+		if (m_WorldAge - m_LastSave > std::chrono::seconds(30))
```

Comment:
Necessary? I would assume that you would want to enforce the cap as quickly as possible.